src/index.js

```diff
@@ -684,7 +842,7 @@ export function interpretQuantity(s) {
 export function splitValue(a) {
 	var i = 0;
 	var a = new BigNumber('' + a);
-	if (a.gte(new BigNumber("10000000000000000")) && a.lt(new BigNumber("100000000000000000000000")) || a.eq(0))
+	if (a.gte(new BigNumber('10000000000000000')) && a.lt(new BigNumber('100000000000000000000000')) || a.eq(0))
```

Comment:
Tiny "optimisation" - it may make sense to have then BN values as constants in this file, i.e. only create them once.